.github/workflows/lima.yaml

```diff
@@ -65,6 +65,14 @@ jobs:
           limactl shell vm1 sudo apt-get update
           limactl shell vm1 sudo apt-get install -y iperf3
           limactl shell vm1 sudo systemctl start iperf3.service
+          echo "Waiting for iperf3 server ..."
+          limactl shell vm1 bash -s << EOF
+            if ! timeout 10s bash -c "until ss -tln | grep -q :5201; do sleep 1; done"; then
```

Comment:
I would have picked a longer timeout, like `30s` just because the GitHub runners are sometimes so sluggish, but maybe this is enough. At least we get a clear error when it fails.